linode/manager
# test: [M3-7123] - Cypress integration test for edit subnet flow

## Description 📝
- added a test for the edit subnet flow

## How to test 🧪
Verify that tests pass: `yarn cy:run -s "cypress/e2e/core/vpc/vpc-details-page.spec.ts"`
